2-advanced/dubbo-samples-group/case-configuration.yml

```diff
@@ -18,7 +18,7 @@ from: app-builtin-zookeeper.yml
 
 props:
   project_name: dubbo-samples-group
-  main_class: org.apache.dubbo.samples.group.GroupProvider
+  main_class: org.apache.dubbo.samples.group.provider.ProviderApplication
```

Comment:
Please refer https://github.com/apache/dubbo-samples/blob/master/2-advanced/dubbo-samples-cache/case-configuration.yml for how to set the right case configuration.